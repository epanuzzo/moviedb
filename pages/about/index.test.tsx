import React from "react";
import renderer from "react-test-renderer";
import AboutText from "./index";

describe("AboutText", () => {
  it("renders the component", () => {
    const tree = renderer.create(<AboutText />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
