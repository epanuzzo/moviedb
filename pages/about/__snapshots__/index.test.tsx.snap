// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutText renders the component 1`] = `
<div
  className="bg-gray-800 text-gray-300 p-6 rounded-lg shadow-lg max-w-3xl mx-auto mt-8 mb-8"
>
  <h2
    className="text-2xl font-bold text-white mb-4"
  >
    About This Website
  </h2>
  <p
    className="leading-relaxed"
  >
    This website uses TMDB and the TMDB APIs but is not endorsed, certified, or otherwise approved by TMDB.
  </p>
  <p
    className="leading-relaxed"
  >
    Welcome to our best 1990 movie database, where you can explore a vast collection of movies from various genres with rating more than 8. Discover detailed information about your favorite films, including cast, director, and other information. Use our search feature to find movies quickly, and enjoy a seamless browsing experience with our mobile-friendly design. Whether you're looking for the latest blockbusters or classic films, we've got something for every movie enthusiast!
  </p>
</div>
`;
